File: aisile-content-interface/src/main/java/com/aisile/content/service/ContentService.java
package com.aisile.content.service;

import java.util.List;

import com.aisile.pojo.TbContent;
import com.aisile.pojo.TbSeller;
import com.aisile.pojo.entity.PageResult;

public interface ContentService {
	
	public List<TbContent> findAll();
	
	public PageResult findPage( int page,int rows);
	
	public PageResult findSearch(int page,int rows,TbContent tbContent);
	
	public void add(TbContent tbContent );
	
	public void update(TbContent tbContent);
	
	public TbContent findOne(Long id); 
	
	public void delete(Long [] ids);

	public void updateStatus(TbSeller tbSeller, String status);
	
	/**
	 * 根据广告类型ID查询列表
	 * @param key
	 * @return
	 */
	public List<TbContent> findByCategoryId(Long categoryId);
	
	//屏蔽
	public void shield(Long [] ids);
	//开启
	public void openq(Long[] ids);

}

File: aisile-manager-web/src/main/webapp/js/service/contentService.js
//封装所有连接后端代码    需要$http
app.service('contentService',function($http){
	//查询全部的数据
	this.findAll = function(){
		return $http.get('../content/findAll.do');
	}
	//分页
	this.findPage = function(page,rows){
		return $http.get('../content/findPage.do?page='+page+"&rows="+rows);
	}
	//分页带数据查询
	this.findSearch = function(page,rows,searchEntity){
		return $http.post('../content/findSearch.do?page='+page+"&rows="+rows,searchEntity);
	}
	//修改
	this.update = function(entity){
		return $http.post('../content/update.do',entity);
	}
	//修改(针对审核使用)
	this.updateStatus = function(entity,status){
		return $http.post('../content/updateStatus.do?status='+status,entity);
	}
	//添加
	this.add = function(entity){
		return $http.post('../content/add.do',entity);
	}
	//根据id查询对象
	this.findOne = function(id){
		return $http.post('../content/findOne.do?id='+id);
	}
	//删除
	this.dele = function(selectIds){
		return $http.get('../content/dele.do?ids='+selectIds);
	}
	//屏蔽
	this.shield = function(selectIds){
		return $http.get('../content/shield.do?ids='+selectIds);
	}
	//开启
	this.openq = function(selectIds){
		return $http.get('../content/openq.do?ids='+selectIds);
	}
	//下拉框
	this.selectOptionList = function(){
		return $http.get('../content/selectOptionList.do');
	}
})